api4/system_test.go

```diff
@@ -571,19 +571,20 @@ func TestClearServerBusy(t *testing.T) {
 	})
 }
 
-func TestGetServerBusyExpires(t *testing.T) {
+func TestGetServerBusy(t *testing.T) {
```

Comment:
Looks like you are replacing the test for the other method with the test for this method, shouldn't be tested both?

Reply:
Yes, definitely.  Fixed via https://github.com/mattermost/mattermost-server/pull/13986/commits/a5ae6983749bae483072d352067d022f11b33b9f   Thanks!